content/hyde-blog-with-netlify-cms.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Hyde starter blog with Netlify CMS"
+github: https://github.com/bdougie/hyde-cms-theme
+author: bdougie
+demo: http://hyde.netlify.com/
+date: 2020-06-03
+ssg:
+    - Hyde
```

Comment:
Hyde will need to be added as a ssg taxonomy

Reply:
Fixed